docs/cookbook.rst

```diff
@@ -0,0 +1,51 @@
+.. Copyright (c) 2019, Lawrence Livermore National Security, LLC. 
+..
+.. Produced at the Lawrence Livermore National Laboratory 
+..
+.. LLNL-CODE-738041.
+..
+.. All rights reserved. 
+..
+.. This file is part of Shroud.
+..
+.. For details about use and distribution, please read LICENSE.
+..
+.. #######################################################################
+
+Cookbook
+========
+
+F_name_impl with fortran_generic
+--------------------------------
+
+Using the *F_name_impl* format string to explicitly name a Fortran
+wrapper when the *fortran_generic* field is used may present some
```

Comment:
The second line here is a bit awkward (at least with the layout in the rst file). 
It took me a few reads to parse:
> ... is used may present..

(It may just be me, but ) please consider revising.

Reply:
I think that line got away from me.